packages/gatsby-plugin-feed/package.json

```diff
@@ -8,9 +8,8 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.0.0",
+    "fs-extra": "^7.0.1",
```

Comment:
This differs from most other packages (most are on 4.x), but that's probably OK.